README.md

```diff
@@ -1,72 +1,204 @@
-# Microsoft Azure SDK for Go
-[![GoDoc](https://godoc.org/github.com/Azure/azure-sdk-for-go?status.svg)](https://godoc.org/github.com/Azure/azure-sdk-for-go) 
+// azure-sdk-for-go provides Go packages for using Azure services
```

Comment:
Delete? I don't think markdown supports non-HTML comments.

Reply:
i'm being cute and including a go-style comment at the top :)

Reply:
Fine with me, maybe move it after the H1?